pandas/core/series.py

```diff
@@ -3039,7 +3041,7 @@ def update(self, other) -> None:
     def sort_values(
         self,
         axis=0,
-        ascending=True,
+        ascending: Union[Union[bool, int], Sequence[Union[bool, int]]] = True,
```

Comment:
Union[bool, Sequence[bool]]. int is no-where in this. same for all the others.

Reply:
and docstring L3062 `ascending : bool, default True` should be updated to match